tests/MongoDB.Driver.Core.Tests/Core/Authentication/ScramSha256AuthenticatorTests.cs

```diff
@@ -61,7 +62,9 @@ public class ScramSha256AuthenticatorTests
             $"r={_clientNonce}{_serverNonce},s={_serverSalt},i={_iterationCount}";
         private static readonly string _clientRequest2 =
             $"c=biws,r={_clientNonce}{_serverNonce},p=dHzbZapWIk4jUhN+Ute9ytag9zjfMHgsqmmiz7AndVQ=";
-        private static readonly string _serverReponse2 = "v=6rriTRBi23WpRR/wtup+mMhUZUn/dB5nLTJRsjl95G4=";
+        private static readonly string _serverResponse2 = "v=6rriTRBi23WpRR/wtup+mMhUZUn/dB5nLTJRsjl95G4=";
+        private static readonly string _clientOptionalFinalRequest = "";
+        private static readonly string _serverOptionalFinalResponse = "";
```

Comment:
As static members should be named:

    __clientOptionalFinalRequest
    __serverOptionalFinalResponse

(with two leading underscores).

Go ahead and fix the 4 above it also.

Reply:
Done